docs/user/guides/setup-single-sign-on-google-email.rst

```diff
@@ -32,7 +40,7 @@ However, you can define which teams users matching your company's domain email a
 
 1. Navigate to the `authorization setting page <https://readthedocs.com/organizations/choose/organization_sso/>`__.
 2. Select **Google** in the :guilabel:`Provider` drop-down.
-3. Specify your Google Workspace domain in the :guilabel:`Domain` input field and press :guilabel:`Save`.
+3. Press :guilabel:`Save`.
```

Comment:
We should make this a bit more clear. I assume the line above "The domain attached to your Google account will be used to match users that sign up with a Google account to your organization." is meant to convey this information? Is there a way to confirm what domain that is?

Reply:
![Screenshot 2023-09-27 at 11-24-55 Authorization - Read the Docs for Business](https://github.com/readthedocs/readthedocs.org/assets/4975310/93c6ee42-c7bf-4cc1-aca9-902ab2fb56fe)

The domain attached to their Google account is shown in the form as a read-only field.

Reply:
Nice, that is good UX 👍